fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -141,7 +169,19 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
             title: '',
             titleLeft: '',
             page: '',
-            arrows: [false, false, false, false]
+            arrows: [false, false, false, false],
+            annunciators: {
+                fmgc: false,
+                fail: false,
+                mcdu_menu: false,
+                fm1: false,
+                ind: false,
+                rdy: false,
+                blank: false,
+                fm2: false,
+            },
+            display_brightness: 0,
+            integral_brightness: 0,
```

Comment:
Maybe these should be camelCase like the other properties? The annunciators are fine if they match the model ones.

Reply:
I don't think uppercase should be a thing in APIs or anything external, but I'm willing to be convinced otherwise 😆 .

Reply:
I don't have a strong opinion on this convention. I just think the naming scheme should be consistent.

Reply:
The problem we have is that all our simvars, model nodes, etc. are snake case, so some stuff cannot be camel-case and it's impossible to be consistent if we do have camel-case in places e.g. the annunciator names in the this API.